app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/sent/SentViewModel.kt

```diff
@@ -28,4 +30,5 @@ import javax.inject.Inject
 @HiltViewModel
 class SentViewModel @Inject constructor() : ViewModel() {
     val transfers = flow<List<Any>> { emit(listOf(1)) }.stateIn(viewModelScope, SharingStarted.Eagerly, null)
+    val selectedTransferIds: SnapshotStateMap<String, Boolean> = mutableStateMapOf<String, Boolean>()
```

Comment:
Don't expose mutable state

Reply:
Yeah there doesn't seem to be any non mutable one